source/features/softwrap-code.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+Soft-wrap code inside codeblocks
```

Comment:
We need a better description because code is already soft-wrapped in most places. Same with the feature name

Reply:
Did my best to explain it, but it still looks a bit bad though!